### PR TITLE
Fix prometheus/promu#65 by compiling only provided ARM platforms

### PR DESCRIPTION
**Pull request by sdurrheimer**
Should be merged just before merging promu's PR

### CODE REVIEW

File: 1.8/powerpc/rootfs/builder.sh [hunk 1]
```diff
@@ -27,12 +28,13 @@ do
   prefix=".build/${goos}-${arch}"
   mkdir -p "${prefix}"
 
-  if [ "${arch}" = "ppc64" ]; then
+  if [[ "${arch}" = "ppc64" ]]; then
```
Comment (SuperQ) [inline 1.8/powerpc/rootfs/builder.sh]: If we're going full-bashism, maybe `==` as well?

File: 1.8/arm/rootfs/builder.sh [hunk 2]
```diff
@@ -17,34 +17,35 @@ source /common.sh
...
+    mkdir -p "${prefix}"
 
-      if [ "${goarm}" = 7 ]; then
-        CC="arm-linux-gnueabihf-gcc" CXX="arm-linux-gnueabihf-g++" GOARM=${goarm} GOOS=${goos} GOARCH=${arch} make PREFIX="${prefix}" build
-      else
-        CC="arm-linux-gnueabi-gcc" CXX="arm-linux-gnueabi-g++" GOARM=${goarm} GOOS=${goos} GOARCH=${arch} make PREFIX="${prefix}" build
-      fi
-    done
-  elif [ "${arch}" = "arm64" ]; then
+    if [ "${goarm}" = 7 ]; then
```
Comment (SuperQ) [inline 1.8/arm/rootfs/builder.sh]: `[[ ]]` here?

**Comment by sdurrheimer** [comment]
Fixed

**Comment by grobie** [comment]
Still WIP?

**Comment by sdurrheimer** [comment]
Nop

**Comment by grobie** [comment]
My bash parser in my head isn't that good, but looks good as far as I can see.

:+1:

**Pull request merged**